app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Users
+      class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+        respond_to :json
+        def google_oauth2
+          @user = User.from_omniauth(auth)
+
+          # TODO: Return JWT token on both success and error
+          if @user.persisted?
+            render_success
```

Comment:
Why extracting to methods when it is a single line of code 😅

Reply:
I was supposed to add more lines of code with generating tokens, so I extracted new methods for future updates 😄